store-implementation/accumulo-store/src/main/java/uk/gov/gchq/gaffer/accumulostore/operation/handler/AddElementsHandler.java

```diff
@@ -18,34 +18,70 @@
 
 import uk.gov.gchq.gaffer.accumulostore.AccumuloStore;
 import uk.gov.gchq.gaffer.data.element.Element;
+import uk.gov.gchq.gaffer.operation.Operation;
 import uk.gov.gchq.gaffer.operation.OperationException;
 import uk.gov.gchq.gaffer.store.Context;
 import uk.gov.gchq.gaffer.store.Store;
 import uk.gov.gchq.gaffer.store.StoreException;
 import uk.gov.gchq.gaffer.store.ValidatedElements;
+import uk.gov.gchq.gaffer.store.operation.handler.FieldDeclaration;
 import uk.gov.gchq.gaffer.store.operation.handler.OperationHandler;
 
-public class AddElementsHandler implements OperationHandler<AddElements> {
-    @Override
-    public Void doOperation(final AddElements operation,
-                            final Context context, final Store store)
-            throws OperationException {
-        addElements(operation, (AccumuloStore) store);
-        return null;
-    }
+public class AddElementsHandler implements OperationHandler<Void> {
+
+    private static final String SKIP_INVALID_ELEMENTS = "skipInvalidElements";
 
-    private void addElements(final AddElements operation, final AccumuloStore store)
-            throws OperationException {
+    private static final String VALIDATE = "validate";
+
+    @Override
+    public Void _doOperation(final Operation operation, final Context context, final Store store) throws OperationException {
         try {
-            final Iterable<?extends Element> validatedElements;
-            if (operation.isValidate()) {
-                validatedElements = new ValidatedElements(operation.input(), store.getSchema(), operation.isSkipInvalidElements());
-            } else {
-                validatedElements = operation.input();
+            final boolean validate = (boolean) operation.getOrDefault(VALIDATE, false);
```

Comment:
`getOrDefault` because you defined it as optional. 
But false needs to match what would have existed as default in the old deleted operation.

Reply:
Looking at the `AddElements` class it should be set to `true`. I have already spotted the error and changed it